packages/compose/src/hooks/use-focus-return/index.js

```diff
@@ -62,7 +64,7 @@ function useFocusReturn( onFocusReturn ) {
 			if ( onFocusReturnRef.current ) {
 				onFocusReturnRef.current();
 			} else {
-				focusedBeforeMount.current.focus();
+				/** @type {HTMLElement} */ ( focusedBeforeMount.current )?.focus();
```

Comment:
Can `focusedBeforeMount.current` be `null` at this point in the code?

```suggestion
				/** @type {null | HTMLElement} */ ( focusedBeforeMount.current )?.focus();
```